include/evmc/helpers.h

```diff
@@ -18,13 +18,29 @@
 #include <evmc/evmc.h>
 
 /**
- * Returns true if the instance has a compatible ABI version.
+ * Returns true if the VM instance has a compatible ABI version.
  */
 static inline int evmc_is_abi_compatible(struct evmc_instance *instance)
 {
     return instance->abi_version == EVMC_ABI_VERSION;
 }
 
+/**
+ * Returns the name of the VM instance.
+ */
+static inline const char* evmc_instance_name(struct evmc_instance *instance)
```

Comment:
Rename it to `evmc_get_vm_name()` or `evmc_vm_get_name()`. The later is better because I want to at least split VM and Host functions with prefixes: `evmc_vm_` and `evmc_host_`.